js/screen/app/ctrls/main_ctrl.js

```diff
@@ -31,53 +65,67 @@ TestApp.controllers.controller('MainCtrl', ['$scope', '$location', '$timeout', f
         checked: false
       }
     ],
+    localisation:[
+      {
+        text: "I tested that the game loads the game in the same language as the main screen (browser or Android TV). If the game does not support the language of the screen, the game defaults to English. NOTE: Only the language of the screen is relevant to the game, not the language of the controllers.",
```

Comment:
Is the note actually necessary? The developer can not decide what language to get the localised data in, that is decided by the AirConsole API before the screen / controller loads.

If it is necessary, the Game Localisation section should point to the localisation guide to ensure that developers use the correct API.